modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -20,11 +20,16 @@ endif::[]
 = Creating a manifest object that includes a customized `br-ex` bridge
 
 ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
-As an alternative to using the `configure-ovs.sh` shell script to set a `br-ex` bridge on a bare-metal platform, you can create a `MachineConfig` object that includes an NMState configuration file. The NMState configuration file creates a customized `br-ex` bridge network configuration on each node in your cluster.
+As an alternative to using the `configure-ovs.sh` shell script to set a `br-ex` bridge on a bare-metal platform, you can create a `MachineConfig` object that includes an NMState configuration file. The `nmstate` API applies the `MachineConfig` object to each node in your cluster.
```

Comment:
> The `nmstate` API applies the `MachineConfig` object to each node in your cluster.

This seems odd.  Technically the YAML API is used, but I don't like the `API` here.   And we apply the config files from the `MachineConfig` object, but not the object itself really.

Maybe:

> The host nmstate service will apply the NMState configuration file.

or


> The host nmstate-configuration.service and nmstate.service will apply the NMState configuration file.

because the nmstate.service and nmstate-configuration.service are systemd and check for files on boot, select based on hostname and apply.